torchinductor/lowering.py

```diff
@@ -22,6 +22,37 @@
 lowerings = {}
 aten = torch.ops.aten
 
+# TODO(jansel): figure out a way generate this automatically
+#               these come as dtype args to kernels
+# based on https://github.com/pytorch/pytorch/blob/9e3eb329df8f701/c10/core/ScalarType.h#L28
+DTYPE_ID_LOOKUP = {
```

Comment:
lol, cc: @ezyang

Reply:
This is fixed on master pytorch

Reply:
@ezyang I definitely still need this on the latest pytorch master.  Example of an op getting `dtype=6` is `aten.new_ones`, but there are many others.

Reply:
oh I bet this is because optionals are not being handled correctly